lib/Listener/AntiSpamReportListener.php

```diff
@@ -63,17 +52,18 @@ public function handle(Event $event): void {
 				return;
 			}
 
-			$email = $this->service->getReportEmail();
-
 			// No anti spam config found
-			if (empty($email)) {
+			if (empty($this->service->getReportEmail())) {
 				return;
 			}
 
 			//Send message to reporting service
-			$this->service->sendSpamReport(
-				$event
-			);
+			try {
+				$message = $this->service->createSpamReportMessageData($event->getAccount(), $event->getMailbox(), $event->getUid());
+				$this->service->sendSpamReport($message);
+			} catch (ServiceException $e) {
+				$this->logger->error($e->getMessage(), [$e]);
```

Comment:
needs the `exception` as key

Reply:
Like so? `[ServiceException::class => $e]`

Reply:
> If an Exception object is passed in the context data, it MUST be in the 'exception' key. Logging exceptions is a common pattern and this allows implementors to extract a stack trace from the exception when the log backend supports it. Implementors MUST still verify that the 'exception' key is actually an Exception before using it as such, as it MAY contain anything.

https://www.php-fig.org/psr/psr-3/

``['exception' => $e]``

Reply:
ah! thanks :)